### PR TITLE
Add wts to allowed analysis_types

### PR DESCRIPTION
**Pull request by ingkebil**
This PR adds `wts` to the whitelist of alllowed analysis_types for creating a case_config for MIP.

Test is part of pr https://github.com/Clinical-Genomics/cg/pull/443

**Review**:
- [x] code review done by @patrikgrenfeldt 
- [ ] ~~test performed by @ingkebil~~
- [ ] deploy approved by @ingkebil 

**Version increment**
This is a patch version. No new functionality was added nor was existing functionality changed.

### CODE REVIEW

File: trailblazer/mip/config.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class SampleSchema(Schema):
     sample_id = fields.Str(required=True)
     analysis_type = fields.Str(
         required=True,
-        validate=validate.OneOf(choices=['wes', 'wgs', 'tga'])
+        validate=validate.OneOf(choices=['wes', 'wgs', 'tga', 'wts'])
```
Comment (patrikgrenfeldt) [inline trailblazer/mip/config.py]: This will allow MIP to run wts samples, shouldn't we have a separate schema for mip-rna?

Comment (ingkebil) [reply trailblazer/mip/config.py]: True that

File: trailblazer/mip/config.py [hunk 2]
```diff
@@ -38,18 +38,47 @@ class ConfigSchema(Schema):
     samples = fields.List(fields.Nested(SampleSchema), required=True)
 
 
+class SampleSchemaRNA(Schema):
+    sample_id = fields.Str(required=True)
+    analysis_type = fields.Str(
+        required=True,
+        validate=validate.OneOf(choices=['wts'])
```
Comment (patrikgrenfeldt) [inline trailblazer/mip/config.py]: future proofed?

Comment (ingkebil) [reply trailblazer/mip/config.py]: rather being lazy

**Pull request merged**